android/java/org/chromium/chrome/browser/onboarding/P3aOnboardingActivity.java

```diff
@@ -126,4 +126,14 @@ public void updateDrawState(@NonNull TextPaint ds) {
 
     @Override
     public void onBackPressed() {}
+
+    @Override
+    protected void triggerLayoutInflation() {
+        setContentView(R.layout.activity_p3a_onboarding);
+    }
+
+    @Override
+    public boolean shouldStartGpuProcess() {
```

Comment:
do we really need to override the default behavior here?

Reply:
The method in super class is abstract so there must be a overridden method. This is what `FirstRunActivity.java` does too.